fiftyone/utils/torch.py

```
@@ -1847,20 +1848,115 @@ def _is_string_array(targets):
         return False
 
 
+def _field_depth(samples, path):
```

Ruff (0.15.14)

[warning] Missing return type annotation for private function `_field_depth`

(ANN202)